modules/virt-what-you-can-do-with-virt.adoc

```diff
@@ -5,9 +5,7 @@
 
 [id="virt-what-you-can-do-with-virt_{context}"]
 = What you can do with {VirtProductName}
-
-{VirtProductName} is an add-on to {product-title} that allows you to run and manage
- virtual machine workloads alongside container workloads.
+{VirtProductName} is a feature of the OpenShift Container Platform that allows you to run and manage virtual machine workloads alongside container workloads.
```

Comment:
Please leave an empty line between the title and the para

Reply:
Empty line added between title and the para.